File: src/sdk-reference/csharp/1/auth/check-token/snippets/check-token.cs
try {
  string jwt = kuzzle.auth.login(
    "local",
    @"{""username"":""foo"",""password"":""bar""}");

  TokenValidity* res = kuzzle.auth.checkToken(jwt);

  if (res.valid)
    Console.WriteLine("Success");
  else
    Console.Error.WriteLine(res.state);
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}

File: src/sdk-reference/cpp/1/auth/check-token/snippets/check-token.cpp
try {
  std::string jwt = kuzzle->auth->login(
    "local",
    R"({"username":"foo","password":"bar"})");

  kuzzleio::token_validity* res = kuzzle->auth->checkToken(jwt);

  if (res->valid)
    std::cout << "Success" << std::endl;
  else
    std::cerr << res->state << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
